src/utils/add.trust.ts

```diff
@@ -53,6 +54,7 @@ const getPageProperties = (
       subtitle: ADD_TRUST_TEXTS.subtitle,
     },
     isUpdate,
+    isFeatureFlagCeaseTrustsEnabled: isActiveFeature(config.FEATURE_FLAG_ENABLE_CEASE_TRUSTS),
```

Comment:
Should this be "FEATURE_FLAG_ENABLE_CEASED_TRUSTS"?

Reply:
I think it used to be called that but has now been renamed to FEATURE_FLAG_ENABLE_CEASE_TRUSTS after previous comments

Reply:
That sounds wrong. Was that one of my comments?

Reply:
I think it was something like FEATURE_FLAG_ENABLE_TRUST_CEASED_DATE but I suggested a new name that was more related to the action of 'ceasing' a trust.